crates/rover-error/src/solution.rs

```diff
@@ -0,0 +1,25 @@
+use houston::HoustonProblem;
+use std::boxed::Box;
+use std::convert::{Into, TryFrom, TryInto};
+use std::error;
+use std::fmt::{self, Display};
+
+#[derive(Debug)]
+pub enum Solution {}
+
+impl Display for Solution {
+    fn fmt(&self, formatter: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(formatter, "no solutions yet...")
+    }
+}
+
+impl<E> TryFrom<E> for Solution
+where
+    E: error::Error + Send + Sync + 'static,
+{
+    type Error = &'static str;
+    fn try_from(error: E) -> Result<Self, Self::Error> {
+        println!("ERrreRROORORR:\n{:?}", &error);
+        Err("Could not find a solution.")
+    }
+}
```

Comment:
this is sorta more what i'm looking to do, but unfortunately I can't wrangle these traits appropriately